### ISSUE TITLE
active_record adapter migration file not included in the gem (gemspec file issue)

### ISSUE THREAD

**Issue by nruth** [jnunemaker/flipper#98]
I think the gem.file matcher for flipper-active_record is leaving out a necessary migration template file.
I propose adding it as a special-case, and cutting a new minor gem version. 

##### Details

On Rails 4.2.4 I'm seeing this error
```
$ bundle exec rails g flipper:active_record
Could not find "migration.rb" in any of your source paths. 
Please invoke Flipper::Generators::ActiveRecordGenerator.source_root(PATH) with the PATH containing your templates. 
Your current source paths are: 
/Users/nruth/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/flipper-active_record-0.7.3/lib/generators/flipper/templates
```

`bundle open flipper-active_record` showed me that the file really is missing; there's no template directory at all.

Cloning the repo and trying out the [gem.file matcher](https://github.com/jnunemaker/flipper/blob/master/flipper-active_record.gemspec#L16) in irb I found it's including these files, which have active_record in their name (as per [the regex in the gemspec](https://github.com/jnunemaker/flipper/blob/master/flipper-active_record.gemspec#L5))

```
docs/active_record/README.md
examples/active_record/basic.rb
examples/active_record/internals.rb
flipper-active_record.gemspec
lib/flipper-active_record.rb
lib/flipper/adapters/active_record.rb
lib/generators/flipper/active_record_generator.rb
spec/flipper/adapters/active_record_spec.rb
test/generators/flipper/active_record_generator_test.rb
lib/flipper/version.rb
```

but not the `lib/generators/flipper/templates/migration.rb`

I guess the simplest fix is to add it in the same way as `lib/flipper/version.rb`?

##### Code
gem file matcher: https://github.com/jnunemaker/flipper/blob/master/flipper-active_record.gemspec#L16
gem file regex: https://github.com/jnunemaker/flipper/blob/master/flipper-active_record.gemspec#L5

**Comment by nruth** [comment]
Another option could be renaming `migration.rb` to `active_record_migration.rb` or something like that.

**Comment by jnunemaker** [comment]
Ah, dang. Good catch. I swear I tested this, but maybe it was using path in the gemfile instead of the released gem.

**Comment by jnunemaker** [comment]
Released 0.7.4 just now. Should be available soon if not already.
